kms/tpmkms/tpmkms.go

```diff
@@ -99,6 +96,137 @@ type TPMKMS struct {
 	identityEarlyRenewalEnabled      bool
 }
 
+// Option is the type used as a variadic argument in NewWithTPM.
+type Option func(o *options) error
```

Comment:
_If_ the `NewWithTPM` function is marked experimental, this probably also should be marked like that.

Reply:
Fixed with 2ef8141